### ISSUE TITLE
Update course level on the card and card label text color

### ISSUE THREAD

**Issue by abdulkdawson** [mitodl/ocw-course-hugo-theme#95]
Old label and level text color
<img width="1148" alt="Screen Shot 2021-05-11 at 2 58 41 PM" src="https://user-images.githubusercontent.com/49685474/117876144-04697200-b271-11eb-8667-43cf2cbde33c.png">

New label and level text color (Hex value should be changed to #3D3D3D)
<img width="656" alt="Screen Shot 2021-05-11 at 3 54 26 PM" src="https://user-images.githubusercontent.com/49685474/117876409-58745680-b271-11eb-99ba-0317bc8bdcfe.png">

Here is the link to the InVision project: [https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/448296968/preview?scrollOffset=498.888916015625](https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/448296968/preview?scrollOffset=498.888916015625)

Card Labels in search
[https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/447781243/preview?scrollOffset=1525.5555419921875](https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/447781243/preview?scrollOffset=1525.5555419921875)

**Comment by pdpinch** [comment]
Fixed in https://github.com/mitodl/ocw-hugo-themes/pull/53
